tests/integration/config.json

```diff
@@ -1,7 +1,7 @@
 {
-    "host": "localhost",
+    "host": "daeunyim-ads-14.postgres.database.azure.com",
     "port": 5432,
-    "user": "postgres",
-    "password": "",
+    "user": "daeunyim",
+    "password": "Demopass123",
```

Comment:
Should these changes be included

Reply:
LOL good catch.

Reply:
I'll need to make another branch and remove this one